src/Contact.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+
+
+const Contact = props => {
+  return (
+    <li className="contact">
+      <div className="image-cropper">
+        <img src={props.avatar} alt="avatar" />
+      </div>
+      <div className="contact-info">
+        <h2>{props.name}</h2>
+        {props.occupation}
+      </div>
+      <button className="delete-button" onClick={
+        event => this.onClick(event)}>X</button>
```

Comment:
You're really close with this delete button!

Instead of passing in the `event`, we really want to pass the contacts `_id`. This means two things for you, first you need to be passing the `_id` as a prop. That is important so we can access it `props._id`.

Next you need to update your function to use the `_id` from props and not the `event`

`onClick={()=> this.onClick(props._id)}`